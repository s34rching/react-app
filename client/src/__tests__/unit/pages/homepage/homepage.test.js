import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../../../pages/homepage/homepage.component';

describe('Pages: Homepage', () => {
  it('should render without errors', () => {
    const wrapper = shallow(<HomePage />);

    expect(wrapper).toMatchSnapshot();
  });
});
