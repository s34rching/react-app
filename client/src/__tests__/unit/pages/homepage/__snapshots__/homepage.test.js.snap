// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages: Homepage should render without errors 1`] = `
<HomepageContainer>
  <Connect(DirectoryMenu) />
</HomepageContainer>
`;
